src/app/components/SkipToContentWrapper.tsx

```diff
@@ -1,6 +1,7 @@
+import { HTMLElement } from '@openstax/types/lib.dom';
 import React, { Component } from 'react';
 import { FormattedMessage } from 'react-intl';
-import scrollTo from '../../helpers/scrollTo';
+import scrollToContent from '../content/components/utils/scrollToContent';
```

Comment:
lost this comment on the first review:

the directory `content` is for book-content related stuff, this helper is an accessibility thing "focus the main content of the page". this should be decoupled from anything happening in the content directory because we'll use it on pages that are not in that module.

the `scrollToContent` helper should be in `app/utils`, and the logic should be decoupled from the knowledge of the navbars, maybe reading the computed `scroll-padding-top` off the html element would be more flexible?

Reply:
Yes, I agree that with the change to the helper to use `getComputedStyle` it can live outside the content dir.